Improve text content on install page (#586)
crystal-lang/crystal-website: _pages/install.md

- '[![](/assets/install/android.svg) #Android](#android)'
- '[![](/assets/install/docker.svg) #Docker](#docker)'
- '[![](/assets/install/construction.svg) #Tools](#developer-tools)'
- '[![](/assets/icons/build-circle.svg) Nightlies](/install/nightlies)'

> - '[![](/assets/icons/source-branch.svg) Source](/install/from_sources)'

Reviewer: the assets are missing. for the nightly I'd rather use a 🌙 like icon

Author: I was trying to find a good icon for nightlies, but couldn't come up with anything great.
If you have a concrete icon in mind that fits into the material set, please let me know.

Reviewer: Let's leave that one for now and fix it later.

---
## Linux

Many Linux distribution have Crystal available in their system packages.
It might not have the most recent version though.

> Third party package managers can be more up to date.

Reviewer: ```suggestion
Third party package managers are typically more up to date.
```
